index.js

```diff
@@ -230,6 +230,7 @@ class Hypercore extends EventEmitter {
 
   async setEncryptionKey (encryptionKey, opts) {
     if (!this.opened) await this.opening
+    if (this.core.unencrypted) return
```

Comment:
should we throw an error?

Reply:
easier to ignore it for autobase atm, but post some engineering, thats prop the way yes